clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AsgConfigHelperSpec.groovy

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Please do not introduce tabs into the codebase.

Reply:
I'd have expected spotless to fix that, I wonder if it's not looking at the tests (or does it not do groovy at all?)

Reply:
@cfieber You are right! `spotlessCheck` did not identify this file as a violator.